src/dashboard/src/main/migrations/0008_fpcommandoutput.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0007_django_upgrade_tweaks'),
+    ]
+
+    operations = [
+        migrations.AlterModelTable(
+            name='fpcommandoutput',
+            table='FPCommandOutput',
+        ),
```

Comment:
Updated this to revert the change pointing FPCommandOutput model at FPCommandOutput table. Modified this migration to move all data from FPCommandOutput table into main_fpcommandoutput table.